code/modules/mob/living/basic/lavaland/lobstrosity/lobstrosity_ai.dm

```diff
@@ -23,13 +24,61 @@
 		/datum/ai_planning_subtree/find_and_hunt_target/lobster_fishing,
 		/datum/ai_planning_subtree/find_fingers,
 	)
+	///The basic mob flee distance key is set to this on New()
+	var/flee_distance = 6
 
-///Ensure that juveline lobstrosities witll charge at things they can reach.
 /datum/ai_controller/basic_controller/lobstrosity/New(atom/new_pawn)
-	if(istype(new_pawn, /mob/living/basic/mining/lobstrosity/juvenile))
-		planning_subtrees -= /datum/ai_planning_subtree/targeted_mob_ability/lobster
-		planning_subtrees += /datum/ai_planning_subtree/targeted_mob_ability/lobster/juvenile
-	return ..()
+	. = ..()
+	set_blackboard_key(BB_BASIC_MOB_FLEE_DISTANCE, flee_distance)
+
+///Ensure that juveline lobstrosities witll charge at things they can reach.
+/datum/ai_controller/basic_controller/lobstrosity/juvenile
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/random_speech/insect,
+		/datum/ai_planning_subtree/hoard_fingers,
+		/datum/ai_planning_subtree/pet_planning,
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/targeted_mob_ability/lobster,
+		/datum/ai_planning_subtree/flee_target/lobster,
+		/datum/ai_planning_subtree/attack_obstacle_in_path,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree/lobster,
+		/datum/ai_planning_subtree/find_food,
+		/datum/ai_planning_subtree/find_and_hunt_target/lobster_fishing,
+		/datum/ai_planning_subtree/find_fingers,
+	)
+	flee_distance = 3
+
+///A subtype of juvenile lobster AI that has the target_retaliate behaviour instead of simple_find_target
+/datum/ai_controller/basic_controller/lobstrosity/juvenile/pacific
```

Comment:
While there are certainly pacific lobsters, do you mean pacifistic?

Reply:
lol